### PR TITLE
Fixed the 3 issues in TorpedoStore

### PR DESCRIPTION
**Pull request by DinuBotan**

### CODE REVIEW

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 1]
```diff
@@ -27,21 +27,21 @@ public TorpedoStore(int numberOfTorpedos){
       }
     }
   }
-
+  Random generator = new Random();
```
Comment (danilashtefan) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:30]: Smart move

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 2]
```diff
@@ -27,21 +27,21 @@ public TorpedoStore(int numberOfTorpedos){
...
 
     // simulate random overheating of the launcher bay which prevents firing
-    Random generator = new Random();
+
     double r = generator.nextDouble();
 
     if (r >= FAILURE_RATE) {
       // successful firing
-      this.torpedoCount =- numberOfTorpedos;
+      this.torpedoCount -= numberOfTorpedos;
```
Comment (danilashtefan) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:44]: Great

**Review by danilashtefan — CHANGES_REQUESTED** [review]
Good job!

**Review by danilashtefan — APPROVED** [review]

**Comment by micskeiz** [comment]
Nice solution, clean repo!

**Pull request merged**